### PR TITLE
Implement individual project pages

### PR DESCRIPTION
**Pull request by mplewis**
Hey look, it's that Bootstrap template! Now with more Tesla Works stuff.

### CODE REVIEW

**Comment by taylortrimble** [comment]
That's so pretty! This opens up into an architecture discussion, but it looks tasty.

**Comment by taylortrimble** [comment]
And for that architecture discussion, jump to #35.

**Comment by taylortrimble** [comment]
> What are the parts you consider necessary for the projects page, btw?

These! (Courtesy Basecamp)
- PM photo and contact info
- Q&A style extended information (cool concept)
- A box for people to type their own suggestions or questions into

Getting people to type into that box will be critical, because it's how we get them to feel like they can add value!

For now, making these post to a Basecamp project is too much. We can just pound a plain-text email to the PM instead, but even that is outside the scope of this PR since it's just layout.

**Comment by mplewis** [comment]
Done:
- PM photo and contact info

Needs to be done:
- Q&A info box
- Suggestions info box

File: templates/project.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+<script>
+// start slideshow
+$(document).ready(function() {
+    $('.slideshow').cycle({
+        fx: 'fade',
+        timeout: 4000
+    });
+});
+</script>
+{% endblock %}
```
Comment (taylortrimble) [inline templates/project.html]: Can we EOF newline?

File: templates/project.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+            <h4><i class="icon-edit"></i> Design</h4>
+            <ul>
+            {% for need in project_data.project_needs.design %}
+                <li>{{ need }}</li>
+            {% endfor %}
+            </ul>
+        </div>
+    </div>
+    <div class="row-fluid">
+        <div class="span6">
```
Comment (taylortrimble) [inline templates/project.html:88]: Can we do this with Jinja column templating and tuples?

Comment (taylortrimble) [inline templates/project.html:88]: That's a little bit of a refactor. Quite so actually. But it's super nice and modular to the point that I prefer. :smile:

File: templates/project.html (hunk 1 above)
Comment (mplewis) [inline templates/project.html]: Done.

File: templates/project.html (hunk 2 above)
Comment (mplewis) [inline templates/project.html:88]: Can you clarify what you mean? I'm missing the point, I think. Redo this as a series of `span6`es without `row-fluid`?

Comment (taylortrimble) [inline templates/project.html:88]: Nope, I like those a lot! Look at this: http://jinja.pocoo.org/docs/templates/#slice

Comment (taylortrimble) [inline templates/project.html:88]: And then attach Headings, icons, and data together with a Tuple.

Comment (mplewis) [inline templates/project.html:88]: From what I'm reading about tuples and slicing, the one thing that I don't like about tuples and slicing is that they make the code hard to read and understand intuitively.

Right now:

```
    "project_needs":
    {
        "engineering":
        [
            "Volunteers to help teach analog theory",
            "Anyone who knows and can help teach PCB layout"
        ],
        ...more stuff here...
    },
```

If we want to make the title of each need modular, how do you feel about:

```
    "project_needs":
    [
        {
            "title": "Engineering",
            "icon": "font-awesome-icon-name",
            "needs":
            [
                "Volunteers to help teach analog theory",
                "Anyone who knows and can help teach PCB layout"
            ]
        },

        ...more stuff here...
    ],
```

**Comment by mplewis** [comment]
Reopening. Lookit.

**Pull request merged**